python3Packages.qrcode: 8.0 -> 8.2

attilaolah opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

attilaolah committed:
python3Packages.qrcode: 8.0 -> 8.1

sarahec commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 403126`

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 16 packages marked as broken and skipped:</summary>
  <ul>
    <li>baserow</li>
    <li>baserow.dist</li>
    <li>froide</li>
    <li>froide-govplan</li>
    <li>froide-govplan.dist</li>
    <li>froide.dist</li>
    <li>gigalixir</li>
    <li>gigalixir.dist</li>
    <li>glitchtip</li>
    <li>netbox_3_7</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>paperless-ngx</li>
    <li>paperless-ngx.dist</li>
    <li>python312Packages.froide</li>
    <li>python312Packages.froide.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>blocksat-cli (python312Packages.blocksat-cli)</li>
    <li>blocksat-cli.dist (python312Packages.blocksat-cli.dist)</li>
    <li>electron-cash</li>
    <li>electron-cash.dist</li>
    <li>electrum</li>
    <li>electrum.dist</li>
    <li>python313Packages.blocksat-cli</li>
    <li>python313Packages.blocksat-cli.dist</li>
    <li>sourcehut.metasrht</li>
    <li>sourcehut.metasrht.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 127 packages built:</summary>
  <ul>
    <li>arubaotp-seed-extractor</li>
    <li>electrum-grs</li>
    <li>electrum-grs.dist</li>
    <li>electrum-ltc</li>
    <li>electrum-ltc.dist</li>
    <li>fdroidserver</li>
    <li>fdroidserver.dist</li>
    <li>lyto</li>
    <li>lyto.dist</li>
    <li>magic-wormhole (python312Packages.magic-wormhole)</li>
    <li>magic-wormhole.dist (python312Packages.magic-wormhole.dist)</li>
    <li>maigret</li>
    <li>maigret.dist</li>
    <li>netbox (netbox_4_2)</li>
    <li>netbox_4_1</li>
    <li>onionshare</li>
    <li>onionshare.dist</li>
    <li>pacu</li>
    <li>pacu.dist</li>
    <li>paperless-asn-qr-codes</li>
    <li>paperless-asn-qr-codes.dist</li>
    <li>pgadmin4</li>
    <li>pgadmin4-desktopmode</li>
    <li>pgadmin4-desktopmode.dist</li>
    <li>pgadmin4.dist</li>
    <li>posteid-seed-extractor</li>
    <li>powerdns-admin</li>
    <li>python312Packages.bilibili-api-python</li>
    <li>python312Packages.bilibili-api-python.dist</li>
    <li>python312Packages.borb</li>
    <li>python312Packages.borb.dist</li>
    <li>python312Packages.dj-rest-auth</li>
    <li>python312Packages.dj-rest-auth.dist</li>
    <li>python312Packages.django-allauth</li>
    <li>python312Packages.django-allauth.dist</li>
    <li>python312Packages.django-mfa3</li>
    <li>python312Packages.django-mfa3.dist</li>
    <li>python312Packages.django-otp</li>
    <li>python312Packages.django-otp-webauthn</li>
    <li>python312Packages.django-otp-webauthn.dist</li>
    <li>python312Packages.django-otp.dist</li>
    <li>python312Packages.django-two-factor-auth</li>
    <li>python312Packages.django-two-factor-auth.dist</li>
    <li>python312Packages.drf-spectacular</li>
    <li>python312Packages.drf-spectacular.dist</li>
    <li>python312Packages.drf-standardized-errors</li>
    <li>python312Packages.drf-standardized-errors.dist</li>
    <li>python312Packages.flask-security</li>
    <li>python312Packages.flask-security.dist</li>
    <li>python312Packages.flet</li>
    <li>python312Packages.flet.dist</li>
    <li>python312Packages.fpdf2</li>
    <li>python312Packages.fpdf2.dist</li>
    <li>python312Packages.netbox-bgp</li>
    <li>python312Packages.netbox-bgp.dist</li>
    <li>python312Packages.netbox-documents</li>
    <li>python312Packages.netbox-documents.dist</li>
    <li>python312Packages.netbox-floorplan-plugin</li>
    <li>python312Packages.netbox-floorplan-plugin.dist</li>
    <li>python312Packages.netbox-plugin-prometheus-sd</li>
    <li>python312Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python312Packages.netbox-reorder-rack</li>
    <li>python312Packages.netbox-reorder-rack.dist</li>
    <li>python312Packages.netbox-routing</li>
    <li>python312Packages.netbox-routing.dist</li>
    <li>python312Packages.pyhanko</li>
    <li>python312Packages.pyhanko.dist</li>
    <li>python312Packages.python-escpos</li>
    <li>python312Packages.python-escpos.dist</li>
    <li>python312Packages.qrcode</li>
    <li>python312Packages.qrcode-terminal</li>
    <li>python312Packages.qrcode-terminal.dist</li>
    <li>python312Packages.qrcode.dist</li>
    <li>python312Packages.reportlab-qrcode</li>
    <li>python312Packages.reportlab-qrcode.dist</li>
    <li>python312Packages.xhtml2pdf</li>
    <li>python312Packages.xhtml2pdf.dist</li>
    <li>python313Packages.bilibili-api-python</li>
    <li>python313Packages.bilibili-api-python.dist</li>
    <li>python313Packages.borb</li>
    <li>python313Packages.borb.dist</li>
    <li>python313Packages.dj-rest-auth</li>
    <li>python313Packages.dj-rest-auth.dist</li>
    <li>python313Packages.django-allauth</li>
    <li>python313Packages.django-allauth.dist</li>
    <li>python313Packages.django-mfa3</li>
    <li>python313Packages.django-mfa3.dist</li>
    <li>python313Packages.django-otp</li>
    <li>python313Packages.django-otp-webauthn</li>
    <li>python313Packages.django-otp-webauthn.dist</li>
    <li>python313Packages.django-otp.dist</li>
    <li>python313Packages.django-two-factor-auth</li>
    <li>python313Packages.django-two-factor-auth.dist</li>
    <li>python313Packages.drf-spectacular</li>
    <li>python313Packages.drf-spectacular.dist</li>
    <li>python313Packages.drf-standardized-errors</li>
    <li>python313Packages.drf-standardized-errors.dist</li>
    <li>python313Packages.flet</li>
    <li>python313Packages.flet.dist</li>
    <li>python313Packages.fpdf2</li>
    <li>python313Packages.fpdf2.dist</li>
    <li>python313Packages.magic-wormhole</li>
    <li>python313Packages.magic-wormhole.dist</li>
    <li>python313Packages.netbox-bgp</li>
    <li>python313Packages.netbox-bgp.dist</li>
    <li>python313Packages.netbox-documents</li>
    <li>python313Packages.netbox-documents.dist</li>
    <li>python313Packages.netbox-plugin-prometheus-sd</li>
    <li>python313Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python313Packages.netbox-reorder-rack</li>
    <li>python313Packages.netbox-reorder-rack.dist</li>
    <li>python313Packages.netbox-routing</li>
    <li>python313Packages.netbox-routing.dist</li>
    <li>python313Packages.pyhanko</li>
    <li>python313Packages.pyhanko.dist</li>
    <li>python313Packages.python-escpos</li>
    <li>python313Packages.python-escpos.dist</li>
    <li>python313Packages.qrcode</li>
    <li>python313Packages.qrcode-terminal</li>
    <li>python313Packages.qrcode-terminal.dist</li>
    <li>python313Packages.qrcode.dist</li>
    <li>python313Packages.reportlab-qrcode</li>
    <li>python313Packages.reportlab-qrcode.dist</li>
    <li>python313Packages.xhtml2pdf</li>
    <li>python313Packages.xhtml2pdf.dist</li>
    <li>social-engineer-toolkit</li>
    <li>tandoor-recipes</li>
  </ul>
</details>

sarahec reviewed (approved):
LGTM on Darwin (built and ran, nixpkgs-review)

attilaolah commented:
I ran `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"` and got no errors, can I do something to help move this forward or are we just waiting on the queued tests? I'm happy to run things locally and can provide x86_64-linux and aarch64-linux.

attilaolah commented:
In the meantime, 8.2 has been released yesterday. I only need 8.1+ so this will version is fine for me, but I'm also happy to update this PR to 8.2 if that'd be better.

sarahec commented:
> In the meantime, 8.2 has been released yesterday. I only need 8.1+ so this will version is fine for me, but I'm also happy to update this PR to 8.2 if that'd be better.

Since this rebuilds quite a few packages, let's incorporate the 8.2 release now.

attilaolah committed:
python3Packages.qrcode: 8.1 -> 8.2

attilaolah commented:
OK I rebased to upstream `master` and updated to 8.2, will run the tests locally now.

attilaolah commented:
Rebuilding 129 packages now, but at least weblate fails to build:

```
       > Checking runtime dependencies for weblate-5.11.1-py3-none-any.whl
       >   - qrcode<8.2,>=7.4.1 not satisfied by version 8.2
```

Other errors don't seem to be related. 3 hours ago, weblate 5.11.3 was released, and the new qrcode dependency is now apparently `>=7.4.1,<8.3`. I'll send a separate PR for weblate.

attilaolah commented:
Once https://github.com/NixOS/nixpkgs/pull/403860 is merged I'll retry the tests. Until that I think we have to wait.

attilaolah commented:
I re-ran `nixpkgs-review`, and I _think_ none of the breakages are due to the version bump here. I think this PR is good to go.

attilaolah commented:
Four of the broken packages are `blocksat-cli`, which I'm attempting to fix in https://github.com/NixOS/nixpkgs/pull/404486.

attilaolah commented:
With current HEAD I get 117 packages that build, and no breakages except for the four that are marked as broken:

```
--------- Report for 'x86_64-linux' ---------
4 packages marked as broken and skipped:
python312Packages.django-mailman3 python312Packages.django-mailman3.dist python313Packages.django-mailman3 python313Packages.django-mailman3.dist
```

I think is is safe to merge now, thanks for the reviews.

nim65s reviewed (approved):
diff lgtm, and is the same as https://nixpkgs-update-logs.nix-community.org/python3Packages.qrcode/2025-05-06.log

Not sure why we fetch from PyPI though: this package also work for me with:
```nix
  src = fetchFromGitHub {
    owner = "lincolnloop";
    repo = "python-qrcode";
    tag = "v${version}";
    hash = "sha256-5KuE3oQcUtQxD1fJeGgoNpKZNizQVuRiZ+vSI+vi7Dc=";
  };
```

Also, maybe, while here, the `with lib;` in meta could be removed.

And this package has no maintainer. Anybody want to volunteer ? I can be added if no one else is motivated :)

attilaolah commented:
I'm also fine maintaining it, seems pretty low-maintenance to me :)

Do we generally prefer fetching from GitHub rather than PyPI for Python packages? (I'm fine with either).

attilaolah commented:
Also, isn't `maintainers` in lib? I see 8707 lines containing `with lib.maintainers` and 14410 containing `with maintainers`, I'd guess since the latter is more popular we would keep the `with lib` as-is?

In any case, I'd be happy to change that but would rather do that in a zero-diff PR to avoid stalling the version bump here.

sarahec commented:
Yes, building from source is preferred (`fetchFromGitHub` on https://github.com/lincolnloop/python-qrcode) as is removing `with lib`. The documentation was just updated yesterday to say that building from source is preferred to fetching from pypi.

attilaolah committed:
python312Packages.qrcode: cleanup meta

attilaolah committed:
python312Packages.qrcode: fetch from GitHub

attilaolah commented:
OK I added myself as maintainer, cleaned up the meta and switched the source to GitHub.

Please give it a review, if it looks good I'll go ahead and update #403127 as well.

nim65s commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 403126`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 15 packages marked as broken and skipped:</summary>
  <ul>
    <li>baserow</li>
    <li>baserow.dist</li>
    <li>froide</li>
    <li>froide-govplan</li>
    <li>froide-govplan.dist</li>
    <li>froide.dist</li>
    <li>mautrix-telegram</li>
    <li>mautrix-telegram.dist</li>
    <li>netbox_3_7</li>
    <li>python312Packages.django-mailman3</li>
    <li>python312Packages.django-mailman3.dist</li>
    <li>python312Packages.froide</li>
    <li>python312Packages.froide.dist</li>
    <li>python313Packages.django-mailman3</li>
    <li>python313Packages.django-mailman3.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>sourcehut.metasrht</li>
    <li>sourcehut.metasrht.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 213 packages built:</summary>
  <ul>
    <li>arubaotp-seed-extractor</li>
    <li>authentik</li>
    <li>blocksat-cli (python312Packages.blocksat-cli)</li>
    <li>blocksat-cli.dist (python312Packages.blocksat-cli.dist)</li>
    <li>blockstream-electrs</li>
    <li>canaille</li>
    <li>canaille.dist</li>
    <li>devtoolbox</li>
    <li>electron-cash</li>
    <li>electron-cash.dist</li>
    <li>electrum</li>
    <li>electrum-grs</li>
    <li>electrum-grs.dist</li>
    <li>electrum-ltc</li>
    <li>electrum-ltc.dist</li>
    <li>electrum.dist</li>
    <li>fdroidserver</li>
    <li>fdroidserver.dist</li>
    <li>freeipa</li>
    <li>gajim</li>
    <li>gajim.dist</li>
    <li>gigalixir</li>
    <li>gigalixir.dist</li>
    <li>glitchtip</li>
    <li>gnome-keysign</li>
    <li>gnome-keysign.dist</li>
    <li>lyto</li>
    <li>lyto.dist</li>
    <li>magic-wormhole (python312Packages.magic-wormhole)</li>
    <li>magic-wormhole.dist (python312Packages.magic-wormhole.dist)</li>
    <li>maigret</li>
    <li>maigret.dist</li>
    <li>mailman-web</li>
    <li>mailman-web.dist</li>
    <li>netbox (netbox_4_2)</li>
    <li>netbox_4_1</li>
    <li>odoo</li>
    <li>odoo.dist</li>
    <li>odoo16</li>
    <li>odoo16.dist</li>
    <li>odoo17</li>
    <li>odoo17.dist</li>
    <li>onionshare</li>
    <li>onionshare-gui</li>
    <li>onionshare-gui.dist</li>
    <li>onionshare.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pacu</li>
    <li>pacu.dist</li>
    <li>paperless-asn-qr-codes</li>
    <li>paperless-asn-qr-codes.dist</li>
    <li>paperless-ngx</li>
    <li>paperless-ngx.dist</li>
    <li>peering-manager</li>
    <li>pgadmin4</li>
    <li>pgadmin4-desktopmode</li>
    <li>pgadmin4-desktopmode.dist</li>
    <li>pgadmin4.dist</li>
    <li>posteid-seed-extractor</li>
    <li>powerdns-admin</li>
    <li>pretalx</li>
    <li>pretalx.dist</li>
    <li>pretalx.static</li>
    <li>pretix</li>
    <li>pretix.dist</li>
    <li>python312Packages.bilibili-api-python</li>
    <li>python312Packages.bilibili-api-python.dist</li>
    <li>python312Packages.borb</li>
    <li>python312Packages.borb.dist</li>
    <li>python312Packages.dj-rest-auth</li>
    <li>python312Packages.dj-rest-auth.dist</li>
    <li>python312Packages.django-allauth</li>
    <li>python312Packages.django-allauth.dist</li>
    <li>python312Packages.django-mfa3</li>
    <li>python312Packages.django-mfa3.dist</li>
    <li>python312Packages.django-otp</li>
    <li>python312Packages.django-otp-webauthn</li>
    <li>python312Packages.django-otp-webauthn.dist</li>
    <li>python312Packages.django-otp.dist</li>
    <li>python312Packages.django-two-factor-auth</li>
    <li>python312Packages.django-two-factor-auth.dist</li>
    <li>python312Packages.drf-spectacular</li>
    <li>python312Packages.drf-spectacular.dist</li>
    <li>python312Packages.drf-standardized-errors</li>
    <li>python312Packages.drf-standardized-errors.dist</li>
    <li>python312Packages.flask-security</li>
    <li>python312Packages.flask-security.dist</li>
    <li>python312Packages.flet</li>
    <li>python312Packages.flet-cli</li>
    <li>python312Packages.flet-cli.dist</li>
    <li>python312Packages.flet-desktop</li>
    <li>python312Packages.flet-desktop.dist</li>
    <li>python312Packages.flet-web</li>
    <li>python312Packages.flet-web.dist</li>
    <li>python312Packages.flet.dist</li>
    <li>python312Packages.fpdf2</li>
    <li>python312Packages.fpdf2.dist</li>
    <li>python312Packages.gdsfactory</li>
    <li>python312Packages.gdsfactory.dist</li>
    <li>python312Packages.netbox-attachments</li>
    <li>python312Packages.netbox-attachments.dist</li>
    <li>python312Packages.netbox-bgp</li>
    <li>python312Packages.netbox-bgp.dist</li>
    <li>python312Packages.netbox-contract</li>
    <li>python312Packages.netbox-contract.dist</li>
    <li>python312Packages.netbox-documents</li>
    <li>python312Packages.netbox-documents.dist</li>
    <li>python312Packages.netbox-floorplan-plugin</li>
    <li>python312Packages.netbox-floorplan-plugin.dist</li>
    <li>python312Packages.netbox-interface-synchronization</li>
    <li>python312Packages.netbox-interface-synchronization.dist</li>
    <li>python312Packages.netbox-napalm-plugin</li>
    <li>python312Packages.netbox-napalm-plugin.dist</li>
    <li>python312Packages.netbox-plugin-prometheus-sd</li>
    <li>python312Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python312Packages.netbox-qrcode</li>
    <li>python312Packages.netbox-qrcode.dist</li>
    <li>python312Packages.netbox-reorder-rack</li>
    <li>python312Packages.netbox-reorder-rack.dist</li>
    <li>python312Packages.netbox-routing</li>
    <li>python312Packages.netbox-routing.dist</li>
    <li>python312Packages.netbox-topology-views</li>
    <li>python312Packages.netbox-topology-views.dist</li>
    <li>python312Packages.nomadnet</li>
    <li>python312Packages.nomadnet.dist</li>
    <li>python312Packages.pyhanko</li>
    <li>python312Packages.pyhanko.dist</li>
    <li>python312Packages.python-escpos</li>
    <li>python312Packages.python-escpos.dist</li>
    <li>python312Packages.qrcode</li>
    <li>python312Packages.qrcode-terminal</li>
    <li>python312Packages.qrcode-terminal.dist</li>
    <li>python312Packages.qrcode.dist</li>
    <li>python312Packages.reportlab-qrcode</li>
    <li>python312Packages.reportlab-qrcode.dist</li>
    <li>python312Packages.xhtml2pdf</li>
    <li>python312Packages.xhtml2pdf.dist</li>
    <li>python313Packages.bilibili-api-python</li>
    <li>python313Packages.bilibili-api-python.dist</li>
    <li>python313Packages.blocksat-cli</li>
    <li>python313Packages.blocksat-cli.dist</li>
    <li>python313Packages.borb</li>
    <li>python313Packages.borb.dist</li>
    <li>python313Packages.dj-rest-auth</li>
    <li>python313Packages.dj-rest-auth.dist</li>
    <li>python313Packages.django-allauth</li>
    <li>python313Packages.django-allauth.dist</li>
    <li>python313Packages.django-mfa3</li>
    <li>python313Packages.django-mfa3.dist</li>
    <li>python313Packages.django-otp</li>
    <li>python313Packages.django-otp-webauthn</li>
    <li>python313Packages.django-otp-webauthn.dist</li>
    <li>python313Packages.django-otp.dist</li>
    <li>python313Packages.django-two-factor-auth</li>
    <li>python313Packages.django-two-factor-auth.dist</li>
    <li>python313Packages.drf-spectacular</li>
    <li>python313Packages.drf-spectacular.dist</li>
    <li>python313Packages.drf-standardized-errors</li>
    <li>python313Packages.drf-standardized-errors.dist</li>
    <li>python313Packages.flet</li>
    <li>python313Packages.flet-cli</li>
    <li>python313Packages.flet-cli.dist</li>
    <li>python313Packages.flet-desktop</li>
    <li>python313Packages.flet-desktop.dist</li>
    <li>python313Packages.flet-web</li>
    <li>python313Packages.flet-web.dist</li>
    <li>python313Packages.flet.dist</li>
    <li>python313Packages.fpdf2</li>
    <li>python313Packages.fpdf2.dist</li>
    <li>python313Packages.gdsfactory</li>
    <li>python313Packages.gdsfactory.dist</li>
    <li>python313Packages.magic-wormhole</li>
    <li>python313Packages.magic-wormhole.dist</li>
    <li>python313Packages.netbox-bgp</li>
    <li>python313Packages.netbox-bgp.dist</li>
    <li>python313Packages.netbox-documents</li>
    <li>python313Packages.netbox-documents.dist</li>
    <li>python313Packages.netbox-interface-synchronization</li>
    <li>python313Packages.netbox-interface-synchronization.dist</li>
    <li>python313Packages.netbox-plugin-prometheus-sd</li>
    <li>python313Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python313Packages.netbox-qrcode</li>
    <li>python313Packages.netbox-qrcode.dist</li>
    <li>python313Packages.netbox-reorder-rack</li>
    <li>python313Packages.netbox-reorder-rack.dist</li>
    <li>python313Packages.netbox-routing</li>
    <li>python313Packages.netbox-routing.dist</li>
    <li>python313Packages.nomadnet</li>
    <li>python313Packages.nomadnet.dist</li>
    <li>python313Packages.pyhanko</li>
    <li>python313Packages.pyhanko.dist</li>
    <li>python313Packages.python-escpos</li>
    <li>python313Packages.python-escpos.dist</li>
    <li>python313Packages.qrcode</li>
    <li>python313Packages.qrcode-terminal</li>
    <li>python313Packages.qrcode-terminal.dist</li>
    <li>python313Packages.qrcode.dist</li>
    <li>python313Packages.reportlab-qrcode</li>
    <li>python313Packages.reportlab-qrcode.dist</li>
    <li>python313Packages.xhtml2pdf</li>
    <li>python313Packages.xhtml2pdf.dist</li>
    <li>seahub</li>
    <li>social-engineer-toolkit</li>
    <li>tahoe-lafs</li>
    <li>tahoe-lafs.dist</li>
    <li>tahoe-lafs.doc</li>
    <li>tahoe-lafs.info</li>
    <li>tandoor-recipes</li>
    <li>warpinator</li>
    <li>weblate</li>
    <li>weblate.dist</li>
    <li>weblate.static</li>
  </ul>
</details>

nim65s reviewed (approved):
sourcehut.metasrht is already broken: https://hydra.nixos.org/build/296485586

siraben commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 403126`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 15 packages marked as broken and skipped:</summary>
  <ul>
    <li>baserow</li>
    <li>baserow.dist</li>
    <li>froide</li>
    <li>froide-govplan</li>
    <li>froide-govplan.dist</li>
    <li>froide.dist</li>
    <li>mautrix-telegram</li>
    <li>mautrix-telegram.dist</li>
    <li>netbox_3_7</li>
    <li>python312Packages.django-mailman3</li>
    <li>python312Packages.django-mailman3.dist</li>
    <li>python312Packages.froide</li>
    <li>python312Packages.froide.dist</li>
    <li>python313Packages.django-mailman3</li>
    <li>python313Packages.django-mailman3.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>canaille</li>
    <li>canaille.dist</li>
    <li>sourcehut.metasrht</li>
    <li>sourcehut.metasrht.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 211 packages built:</summary>
  <ul>
    <li>arubaotp-seed-extractor</li>
    <li>authentik</li>
    <li>blocksat-cli (python312Packages.blocksat-cli)</li>
    <li>blocksat-cli.dist (python312Packages.blocksat-cli.dist)</li>
    <li>blockstream-electrs</li>
    <li>devtoolbox</li>
    <li>electron-cash</li>
    <li>electron-cash.dist</li>
    <li>electrum</li>
    <li>electrum-grs</li>
    <li>electrum-grs.dist</li>
    <li>electrum-ltc</li>
    <li>electrum-ltc.dist</li>
    <li>electrum.dist</li>
    <li>fdroidserver</li>
    <li>fdroidserver.dist</li>
    <li>freeipa</li>
    <li>gajim</li>
    <li>gajim.dist</li>
    <li>gigalixir</li>
    <li>gigalixir.dist</li>
    <li>glitchtip</li>
    <li>gnome-keysign</li>
    <li>gnome-keysign.dist</li>
    <li>lyto</li>
    <li>lyto.dist</li>
    <li>magic-wormhole (python312Packages.magic-wormhole)</li>
    <li>magic-wormhole.dist (python312Packages.magic-wormhole.dist)</li>
    <li>maigret</li>
    <li>maigret.dist</li>
    <li>mailman-web</li>
    <li>mailman-web.dist</li>
    <li>netbox (netbox_4_2)</li>
    <li>netbox_4_1</li>
    <li>odoo</li>
    <li>odoo.dist</li>
    <li>odoo16</li>
    <li>odoo16.dist</li>
    <li>odoo17</li>
    <li>odoo17.dist</li>
    <li>onionshare</li>
    <li>onionshare-gui</li>
    <li>onionshare-gui.dist</li>
    <li>onionshare.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pacu</li>
    <li>pacu.dist</li>
    <li>paperless-asn-qr-codes</li>
    <li>paperless-asn-qr-codes.dist</li>
    <li>paperless-ngx</li>
    <li>paperless-ngx.dist</li>
    <li>peering-manager</li>
    <li>pgadmin4</li>
    <li>pgadmin4-desktopmode</li>
    <li>pgadmin4-desktopmode.dist</li>
    <li>pgadmin4.dist</li>
    <li>posteid-seed-extractor</li>
    <li>powerdns-admin</li>
    <li>pretalx</li>
    <li>pretalx.dist</li>
    <li>pretalx.static</li>
    <li>pretix</li>
    <li>pretix.dist</li>
    <li>python312Packages.bilibili-api-python</li>
    <li>python312Packages.bilibili-api-python.dist</li>
    <li>python312Packages.borb</li>
    <li>python312Packages.borb.dist</li>
    <li>python312Packages.dj-rest-auth</li>
    <li>python312Packages.dj-rest-auth.dist</li>
    <li>python312Packages.django-allauth</li>
    <li>python312Packages.django-allauth.dist</li>
    <li>python312Packages.django-mfa3</li>
    <li>python312Packages.django-mfa3.dist</li>
    <li>python312Packages.django-otp</li>
    <li>python312Packages.django-otp-webauthn</li>
    <li>python312Packages.django-otp-webauthn.dist</li>
    <li>python312Packages.django-otp.dist</li>
    <li>python312Packages.django-two-factor-auth</li>
    <li>python312Packages.django-two-factor-auth.dist</li>
    <li>python312Packages.drf-spectacular</li>
    <li>python312Packages.drf-spectacular.dist</li>
    <li>python312Packages.drf-standardized-errors</li>
    <li>python312Packages.drf-standardized-errors.dist</li>
    <li>python312Packages.flask-security</li>
    <li>python312Packages.flask-security.dist</li>
    <li>python312Packages.flet</li>
    <li>python312Packages.flet-cli</li>
    <li>python312Packages.flet-cli.dist</li>
    <li>python312Packages.flet-desktop</li>
    <li>python312Packages.flet-desktop.dist</li>
    <li>python312Packages.flet-web</li>
    <li>python312Packages.flet-web.dist</li>
    <li>python312Packages.flet.dist</li>
    <li>python312Packages.fpdf2</li>
    <li>python312Packages.fpdf2.dist</li>
    <li>python312Packages.gdsfactory</li>
    <li>python312Packages.gdsfactory.dist</li>
    <li>python312Packages.netbox-attachments</li>
    <li>python312Packages.netbox-attachments.dist</li>
    <li>python312Packages.netbox-bgp</li>
    <li>python312Packages.netbox-bgp.dist</li>
    <li>python312Packages.netbox-contract</li>
    <li>python312Packages.netbox-contract.dist</li>
    <li>python312Packages.netbox-documents</li>
    <li>python312Packages.netbox-documents.dist</li>
    <li>python312Packages.netbox-floorplan-plugin</li>
    <li>python312Packages.netbox-floorplan-plugin.dist</li>
    <li>python312Packages.netbox-interface-synchronization</li>
    <li>python312Packages.netbox-interface-synchronization.dist</li>
    <li>python312Packages.netbox-napalm-plugin</li>
    <li>python312Packages.netbox-napalm-plugin.dist</li>
    <li>python312Packages.netbox-plugin-prometheus-sd</li>
    <li>python312Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python312Packages.netbox-qrcode</li>
    <li>python312Packages.netbox-qrcode.dist</li>
    <li>python312Packages.netbox-reorder-rack</li>
    <li>python312Packages.netbox-reorder-rack.dist</li>
    <li>python312Packages.netbox-routing</li>
    <li>python312Packages.netbox-routing.dist</li>
    <li>python312Packages.netbox-topology-views</li>
    <li>python312Packages.netbox-topology-views.dist</li>
    <li>python312Packages.nomadnet</li>
    <li>python312Packages.nomadnet.dist</li>
    <li>python312Packages.pyhanko</li>
    <li>python312Packages.pyhanko.dist</li>
    <li>python312Packages.python-escpos</li>
    <li>python312Packages.python-escpos.dist</li>
    <li>python312Packages.qrcode</li>
    <li>python312Packages.qrcode-terminal</li>
    <li>python312Packages.qrcode-terminal.dist</li>
    <li>python312Packages.qrcode.dist</li>
    <li>python312Packages.reportlab-qrcode</li>
    <li>python312Packages.reportlab-qrcode.dist</li>
    <li>python312Packages.xhtml2pdf</li>
    <li>python312Packages.xhtml2pdf.dist</li>
    <li>python313Packages.bilibili-api-python</li>
    <li>python313Packages.bilibili-api-python.dist</li>
    <li>python313Packages.blocksat-cli</li>
    <li>python313Packages.blocksat-cli.dist</li>
    <li>python313Packages.borb</li>
    <li>python313Packages.borb.dist</li>
    <li>python313Packages.dj-rest-auth</li>
    <li>python313Packages.dj-rest-auth.dist</li>
    <li>python313Packages.django-allauth</li>
    <li>python313Packages.django-allauth.dist</li>
    <li>python313Packages.django-mfa3</li>
    <li>python313Packages.django-mfa3.dist</li>
    <li>python313Packages.django-otp</li>
    <li>python313Packages.django-otp-webauthn</li>
    <li>python313Packages.django-otp-webauthn.dist</li>
    <li>python313Packages.django-otp.dist</li>
    <li>python313Packages.django-two-factor-auth</li>
    <li>python313Packages.django-two-factor-auth.dist</li>
    <li>python313Packages.drf-spectacular</li>
    <li>python313Packages.drf-spectacular.dist</li>
    <li>python313Packages.drf-standardized-errors</li>
    <li>python313Packages.drf-standardized-errors.dist</li>
    <li>python313Packages.flet</li>
    <li>python313Packages.flet-cli</li>
    <li>python313Packages.flet-cli.dist</li>
    <li>python313Packages.flet-desktop</li>
    <li>python313Packages.flet-desktop.dist</li>
    <li>python313Packages.flet-web</li>
    <li>python313Packages.flet-web.dist</li>
    <li>python313Packages.flet.dist</li>
    <li>python313Packages.fpdf2</li>
    <li>python313Packages.fpdf2.dist</li>
    <li>python313Packages.gdsfactory</li>
    <li>python313Packages.gdsfactory.dist</li>
    <li>python313Packages.magic-wormhole</li>
    <li>python313Packages.magic-wormhole.dist</li>
    <li>python313Packages.netbox-bgp</li>
    <li>python313Packages.netbox-bgp.dist</li>
    <li>python313Packages.netbox-documents</li>
    <li>python313Packages.netbox-documents.dist</li>
    <li>python313Packages.netbox-interface-synchronization</li>
    <li>python313Packages.netbox-interface-synchronization.dist</li>
    <li>python313Packages.netbox-plugin-prometheus-sd</li>
    <li>python313Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python313Packages.netbox-qrcode</li>
    <li>python313Packages.netbox-qrcode.dist</li>
    <li>python313Packages.netbox-reorder-rack</li>
    <li>python313Packages.netbox-reorder-rack.dist</li>
    <li>python313Packages.netbox-routing</li>
    <li>python313Packages.netbox-routing.dist</li>
    <li>python313Packages.nomadnet</li>
    <li>python313Packages.nomadnet.dist</li>
    <li>python313Packages.pyhanko</li>
    <li>python313Packages.pyhanko.dist</li>
    <li>python313Packages.python-escpos</li>
    <li>python313Packages.python-escpos.dist</li>
    <li>python313Packages.qrcode</li>
    <li>python313Packages.qrcode-terminal</li>
    <li>python313Packages.qrcode-terminal.dist</li>
    <li>python313Packages.qrcode.dist</li>
    <li>python313Packages.reportlab-qrcode</li>
    <li>python313Packages.reportlab-qrcode.dist</li>
    <li>python313Packages.xhtml2pdf</li>
    <li>python313Packages.xhtml2pdf.dist</li>
    <li>seahub</li>
    <li>social-engineer-toolkit</li>
    <li>tahoe-lafs</li>
    <li>tahoe-lafs.dist</li>
    <li>tahoe-lafs.doc</li>
    <li>tahoe-lafs.info</li>
    <li>tandoor-recipes</li>
    <li>warpinator</li>
    <li>weblate</li>
    <li>weblate.dist</li>
    <li>weblate.static</li>
  </ul>
</details>
